go-update/src/test/java/org/reactome/release/goupdate/JavaVsPerlComparison.java

```diff
@@ -21,58 +26,98 @@
 	{
-		MySQLAdaptor perlUpdatedDB = new MySQLAdaptor("localhost", "gk_central_2019_04_18", "root", "root", 3306);//new MySQLAdaptor("localhost", "gk_central_after_uniprot_update.dump.sql", "root", "root", 3308);
-		MySQLAdaptor javaUpdatedDB = new MySQLAdaptor("localhost", "gk_central_2019_04_18", "root", "root", 3307);//new MySQLAdaptor("localhost", "gk_central_R66_before_chebi_update.sql", "root", "root", 3307);
+		MySQLAdaptor perlUpdatedDB = new MySQLAdaptor("localhost", "perl_gk_central", "root", "root", 3306);
+		MySQLAdaptor javaUpdatedDB = new MySQLAdaptor("localhost", "java_gk_central", "root", "root", 3306);
+
+		Map<String, Integer> diffNameToCount = new ConcurrentHashMap<>();
+		Map<String, Integer> sameNameToCount = new ConcurrentHashMap<>();
+
 		AtomicInteger diffCount = new AtomicInteger(0);
 		AtomicInteger sameCount = new AtomicInteger(0);
-		
+
 		AtomicInteger regulationDiffCount = new AtomicInteger(0);
 		AtomicInteger regulationSameCount = new AtomicInteger(0);
-		
+
 		AtomicInteger peDiffCount = new AtomicInteger(0);
 		AtomicInteger peSameCount = new AtomicInteger(0);
-		
+
 		AtomicInteger catActDiffCount = new AtomicInteger(0);
 		AtomicInteger catActSameCount = new AtomicInteger(0);
-		
+
 		AtomicInteger accessionsMissingFromPerl = new AtomicInteger(0);
-		
+
 		@SuppressWarnings("unchecked")
-		Set<GKInstance> javaUpdatedGOBiologicalProcesses = (Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_BiologicalProcess);
+		Set<GKInstance> javaUpdatedGOBiologicalProcesses =
+			(Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_BiologicalProcess);
 		@SuppressWarnings("unchecked")
-		Set<GKInstance> javaUpdatedGOMolecularFunctions = (Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_MolecularFunction);
+		Set<GKInstance> javaUpdatedGOMolecularFunctions =
+			(Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_MolecularFunction);
 		@SuppressWarnings("unchecked")
-		Set<GKInstance> javaUpdatedGOCellularComponents = (Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_CellularComponent);
-		
+		Set<GKInstance> javaUpdatedGOCellularComponents =
+			(Set<GKInstance>) javaUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_CellularComponent);
+
+		System.out.println("All Java GO instances obtained");
+
 		List<GKInstance> listOfAllJavaGOThings = new ArrayList<>();
 		listOfAllJavaGOThings.addAll(javaUpdatedGOCellularComponents);
 		listOfAllJavaGOThings.addAll(javaUpdatedGOMolecularFunctions);
 		listOfAllJavaGOThings.addAll(javaUpdatedGOBiologicalProcesses);
-		
+
+		@SuppressWarnings("unchecked")
+		Set<GKInstance> perlUpdatedGOBiologicalProcesses =
+			(Set<GKInstance>) perlUpdatedDB.fetchInstancesByClass(ReactomeJavaConstants.GO_BiologicalProcess);
+		@SuppressWarnings("unchecked")
+		Set<GKInstance> perlUpdatedGOMolecularFunctions = (
```

Comment:
_Nitpick alert_: The formatting differs between line 67-74 when initializing the Sets. Lines 67 & 73 do not have the opening parentheses but line 70 does

Reply:
Nitpick picked :): https://github.com/reactome/data-release-pipeline/pull/121/commits/ce8ce4883a21048b4e003618de3da7b8ba3ae538#diff-a1b0b46e16fc2dac4e77db0ea04b1bb7R74